app/resources/v1/pacbio/library_resource.rb

```diff
@@ -32,6 +31,12 @@ def self.default_sort
         [{ field: 'created_at', direction: :desc }]
       end
 
+      def self.records(_options = {})
+        # We only want to include only libraries without pools
+        # Libraries with pools should be referenced via the LibraryPoolResource
+        _model_class.where(pool: nil)
```

Comment:
I looked in other places where we have done similar and we use model_name (not sure if it is needed) and then super e.g. https://github.com/sanger/traction-service/blob/develop/app/resources/v1/pacbio/plate_resource.rb#L21. I wonder if this would be better than using _model_class as that is an internal method. We could also use a scope but it feels like there is no need for that as it is a temporary method.

Reply:
Good spot, I have updated it.